backend/app.py

```
@@ -37,6 +38,13 @@
 app.config["PROJECT_ROOT"] = os.path.abspath(
     os.path.join(os.path.dirname(__file__), ".."))
 
+# Set secret key (required for sessions and can be used for other cryptographic needs)
+app.config["SECRET_KEY"] = os.environ.get("SECRET_KEY", "dev-secret-key")
+if app.config["SECRET_KEY"] == "dev-secret-key" and not app.config.get(
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)